src/main/java/com/salesforce/cdp/queryservice/util/QueryExecutor.java

```diff
@@ -54,6 +57,12 @@ public QueryExecutor(QueryServiceConnection connection, OkHttpClient tokenClient
         this.queryClient = updateClientWithSocketFactory(client, connection.isSocksProxyDisabled());
     }
 
+    QueryExecutor(QueryServiceConnection connection, OkHttpClient tokenClient, OkHttpClient client, TokenManager tokenManager) {
```

Comment:
When would we use the other constructors? That would lead to error right?

Reply:
nvm, in that case we are initialising TokenManager. Then let's remove the duplicate code lines from the constructor.